static/js/narrow.js

```diff
@@ -749,6 +749,29 @@ exports.restore_home_state = function () {
     navigate.maybe_scroll_to_selected();
 };
 
+function show_search_query() {
+    // Don't need to handle "search:" filter because search_query does not contain it.
+    var search_query = $("#search_query").val();
```

Comment:
Can we get this from something like `current_filter`, so that it's already pre-processed?